tests/dapr_tests.mk

```diff
@@ -629,7 +629,7 @@ setup-kind:
 	# Connect the registry to the KinD network.
 	docker network connect "kind" kind-registry
 	# Setup metrics-server
-	helm install ms stable/metrics-server -n kube-system --set=args={--kubelet-insecure-tls}
+	helm upgrade --install metrics-server metrics-server/metrics-server --namespace kube-system --set=args={--kubelet-insecure-tls}
```

Comment:
We're changing the release name here, will that impact anything?

Reply:
Not that I'm aware of. This is made for developers who wish to run tests locally, there's no part of the code that I'm familiar with that interacts with the metrics server by it's helm release name.
The command is taken from the official metrics-server helm chart website
https://artifacthub.io/packages/helm/metrics-server/metrics-server#installing-the-chart